### DISCUSSION TITLE
Embedded covers not saved in files

### DISCUSSION THREAD

**Discussion by ggeorgo** [strawberrymusicplayer/strawberry#1222] [Q&A]
I have my music (only mp3) located in a NAS folder, and I am accessing it mostly from my main system, Manjaro KDE.
I am editing all my song with correct tags and covers. Most of them work, except some covers, which are not saved as embedded. 
Other tags, are saved, and I have checked multiple times that I do have access and files are not read-only.
Not sure if this is happening only when using internal browser for covers, or when I load a local file.
Also, many times, a song has already a cover, but I want to change it, and instead I see the default (blank).

Any ideas why this is happening, or how to troubleshoot it?

Qt 6.5.0
Strawberry 1.0.17
Kernel 6.1.31

**Comment by jonaski** [comment]
1.0.17 only supported JPEG as embedded image, should be mostly fixed in 1.0.18. Are you saving covers from the tag editor. the cover manager or the search?

**Reply by ggeorgo** [reply]
I am using the tag editor. 
Yesterday I tried saving the cover and load it from disk, and worked in the same file.
